examples/rxJavaExample/src/main/java/io/realm/examples/rxjava/animation/AnimationActivity.java

```diff
@@ -55,23 +57,24 @@ protected void onResume() {
         // Load all persons and start inserting them with 1 sec. intervals.
         // All RealmObject access has to be done on the same thread `findAllAsync` was called on.
         // Warning: This example doesn't handle back pressure well.
-        subscription = realm.where(Person.class).findAllAsync().asObservable()
-                .flatMap(new Func1<RealmResults<Person>, Observable<Person>>() {
+        disposable = realm.where(Person.class).findAllAsync().asFlowable()
+                .flatMap(new Function<RealmResults<Person>, Publisher<Person>>() {
```

Comment:
You know, I've actually just realized that with [Android build tools 3.0](https://github.com/realm/realm-java/blob/b3e2cb841dfacd7e054324bf5028cfc90f5aa7b5/realm/build.gradle#L12), lambda support is out of the box.

I'll try to give a mention to @cmelchior to see what he thinks about converting these Functions and Predicates to lambda expressions instead? Although you still need to tell Gradle that you're actually Java 8 language compatible.

Reply:
Converting to Java 8 is fine with me 👍  Would improve the readability quite a lot for this.